source/user-manual/reference/ossec-conf/rule_test.rst

```diff
@@ -0,0 +1,85 @@
+.. Copyright (C) 2020 Wazuh, Inc.
+
+.. _reference_ossec_rule_test:
+
+rule_test
+=========
+
+.. topic:: XML section name
+
+	.. code-block:: xml
+
+		<rule_test>
+		</rule_test>
+
+Here is how to configure the wazuh-logtest module. It allows to test rules and decoders from Wazuh API and Kibana through Wazuh app 
```

Comment:
Warning. wazuh-logtest may confuse with the tool. In this case we are talking about the whole Wazuh Logtest solution